boards/arm/npcx7m6fb_evb/doc/index.rst

```diff
@@ -79,10 +79,20 @@ JTAG only sessions.
 Flashing
 ========
 
-Build application as usual for the ``npcx7m6fb_evb`` board, and flash
-using Servo V2, μServo, or Servo V4 (CCD). See the
+If the correct headers are installed, this board supports both J-TAG and also
+the ChromiumOS servo.
+
+To flash using Servo V2, μServo, or Servo V4 (CCD), see the
 `Chromium EC Flashing Documentation`_ for more information.
 
+To flash with J-TAG, install the drivers for your programmer, for example:
+SEGGER J-link's drivers are at https://www.segger.com/downloads/jlink/
+
+Build and flash the blinky sample.::
+
+  west build -t clean && \
+    west build -c -p auto -b npcx7m6fb_evb samples/basic/blinky && \
```

Comment:
you may want to pick `hello_world` here, this board does not have an led set for the blinky sample :-) (or define an blinky LED I guess, GPIOC0 I think)

Reply:
You're right. I will pick `hello_world` instead.

Reply:
Done.